docs/mkdocs.yml

```diff
@@ -78,6 +78,8 @@ pages:
   - Triggers: APIGuide/Triggers.md
   - Metrics:  APIGuide/Metrics.md
   - ML Pipeline: APIGuide/MLPipeline/DLEstimator_DLClassifier.md
+  - Support Caffe Layers: APIGuide/caffe_layer_list.md
+  - Support Tensorflow Operations: APIGuide/tensorflow_ops_list.md
```

Comment:
Do we need to create an entry on the doc website? Or a link to the supported list from `caffe-support` or `tensorflow-support` is good enough?

Reply:
we need to create it

Reply:
OK. I will add keras as well.